tasks/move-granules/index.js

```diff
@@ -218,17 +218,20 @@ async function moveGranuleFiles(granulesObject, sourceBucket) {
 *
 * @param {string} cmrFiles - array of objects that include CMR xmls uris and granuleIds
 * @param {Object} granulesObject - an object of the granules where the key is the granuleId
+ * @param {string} distEndpoint - the api distribution endpoint
 * @returns {Promise} promise resolves when all files have been updated
 **/
-async function updateCmrFileAccessURls(cmrFiles, granulesObject) {
+async function updateCmrFileAccessURls(cmrFiles, granulesObject, distEndpoint) {
```

Comment:
I think this may be fixing a mistake I made 😅, but the last `l` in `updateCmrFileAccessURls` should probably be uppercase `updateCmrFileAccessURLs`